README.md

```diff
@@ -6,26 +8,114 @@ An awesome tool to develop and test your micro-frontend services !
 
 Install
 -------
-`$ npm install -g @capsulajs/capsula-hub`
-
-or
-
-`$ yarn global add @capsulajs/capsula-hub`
+In your project, run `$ npm install @capsulajs/capsula-hub` or `$ yarn add @capsulajs/capsula-hub`.
+If you want to use `capsula-hub` command directly, you can install it globally.
 
 Usage
 -----
-`capsula-hub run` if you install it globally
-`yarn/npm run` with npm scripts
+`capsula-hub run -l <path/to/configuration_file>`
```

Comment:
Should it be a relative or an absolute link?

Reply:
Relative